README.md

```diff
@@ -155,7 +155,6 @@ redis-cli DEL boost-relay/sepolia:validators-registration boost-relay/sepolia:va
 * `DISABLE_PAYLOAD_DATABASE_STORAGE` - builder API - disable storing execution payloads in the database (i.e. when using memcached as data availability redundancy)
 * `DISABLE_LOWPRIO_BUILDERS` - reject block submissions by low-prio builders
 * `FORCE_GET_HEADER_204` - force 204 as getHeader response
-* `MEMCACHE_ALLOW_SAVING_FAIL` -- don't abort builder submission if memcache saving fails
```

Comment:
Why was this removed? Is it because we were able to resolve the memcached connectivity issues on deploys?